utils/src/test/kotlin/OrtAuthenticatorTest.kt

```diff
@@ -61,6 +61,19 @@ class OrtAuthenticatorTest : WordSpec({
             authentication.password shouldBe "bar".toCharArray()
         }
 
+        "machine statement precedence" {
```

Comment:
Please name test cases to that they form a sentence with the outer text. Here the outer text is "getNetrcAuthentication() should ..." but "machine statement precedence" does not complete that sentence in a meaningful / grammatically correct way.

Reply:
Proposal: "prefer machine-specific entries over the default"